Exiled.API/Features/Player.cs

```diff
@@ -2694,6 +2694,21 @@ public void EnableEffect(EffectType type, float duration = 0f, bool addDurationI
                 EnableEffect(statusEffect, duration, addDurationIfActive);
         }
 
+        /// <summary>
+        /// Enables a <see cref="Effect">status effect</see> on the player.
+        /// </summary>
+        /// <param name="effect">The <see cref="Effect"/> to enable.</param>
+        public void EnableEffect(Effect effect)
+        {
+            if (effect.IsEnabled)
+            {
+                EnableEffect(effect.Type, effect.Duration, effect.AddDurationIfActive);
+
+                if (effect.Intensity > 0)
```

Comment:
this should be > 1 no ?

Reply:
its > 0 for not errors because intensity begins with 1